.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,29 @@
+# This runs the integration tests for kuberpult
+name: "Kuberpult Integration Tests"
+on:
+  workflow_call:
+    inputs:
+      trigger:
+        required: true
+        type: string
+        description: 'The trigger which calls the workflow, one of [pull-request, main, tag]'
+jobs:
+  integration-test:
+    runs-on: ubuntu-latest
+    steps: 
+    - name: Checkout repository
+      uses: actions/checkout@v3.1.0
+      with:
+        fetch-depth: 0
+    - name: Create bare repository
+      run: |
+        mkdir -p ./services/cd-service/repository_remote
+        git -C ./services/cd-service/repository_remote init --bare
+        git clone ./services/cd-service/repository_remote ./services/cd-service/repository_checkedout
+    - name: Run docker compose
+      run: |
+        IMAGE_REGISTRY=europe-west3-docker.pkg.dev/fdc-public-docker-registry/kuberpult \
+        IMAGE_TAG=$(make version) \
+        docker compose -f ./docker-compose.tpl.yml up -d
```

Comment:
where are the template values replaced by real values? Does this just happen by having the env vars set?

Reply:
Yes! 😄 
I tested it locally and docker compose can use env vars implicitly

Reply:
ok, nice :D I did not know